releases/release-1.32/release-notes/release-notes-draft.json

```diff
@@ -1724,6 +1857,50 @@
       "cluster-lifecycle"
     ]
   },
+  "126953": {
+    "commit": "6568b4bdb39b17105cd31b32899f687a346ce3d7",
+    "text": "kubeadm: removed preflight check for existence of the conntrack binary, as conntrack is no longer a kube-proxy dependency in version 1.32 and newer.",
+    "markdown": "Kubeadm: removed preflight check for existence of the conntrack binary, as conntrack is no longer a kube-proxy dependency in version 1.32 and newer. ([#126953](https://github.com/kubernetes/kubernetes/pull/126953), [@aroradaman](https://github.com/aroradaman)) [SIG Cluster Lifecycle]",
+    "author": "aroradaman",
+    "author_url": "https://github.com/aroradaman",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/126953",
+    "pr_number": 126953,
+    "areas": [
+      "kubeadm"
+    ],
+    "kinds": [
+      "cleanup"
+    ],
+    "sigs": [
+      "cluster-lifecycle"
+    ]
+  },
+  "126957": {
+    "commit": "3d65369f346d359bcabdc7f04addff5fdb2f194a",
+    "text": "Fixed a kubelet and kube-apiserver memory leak in default 1.29 configurations related to tracing.",
+    "markdown": "Fixes a kubelet and kube-apiserver memory leak in default 1.29 configurations related to tracing. ([#126957](https://github.com/kubernetes/kubernetes/pull/126957), [@dashpole](https://github.com/dashpole)) [SIG API Machinery, Architecture, Instrumentation and Node]",
+    "author": "dashpole",
+    "author_url": "https://github.com/dashpole",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/126957",
+    "pr_number": 126957,
+    "areas": [
+      "kubelet",
+      "apiserver",
+      "dependency"
+    ],
+    "kinds": [
+      "bug",
+      "regression"
+    ],
+    "sigs": [
+      "api-machinery",
+      "architecture",
+      "instrumentation",
+      "node"
+    ],
+    "duplicate": true,
+    "duplicate_kind": true
+  },
   "126961": {
     "commit": "e74205b08ad15a2674d1fbaaed98469e882858c7",
     "text": "The CSI volume plugin stopped watching the VolumeAttachment object if the object is not found or the volume is not attached when kubelet waits for a volume attached. In the past, it would fail due to missing permission.",
```

Comment:
```suggestion
    "text": "The CSI volume plugin stopped watching the `VolumeAttachment` object if the object is not found or the volume is not attached when kubelet waits for a volume attached. In the past, it would fail due to the missing of proper permission.",
```